source/overview/license-and-subscription.rst

```diff
@@ -0,0 +1,138 @@
+Licensing and Subscription
+----------------------
+
+Mattermost offers a `secure self-service Customer Portal <https://customers.mattermost.com>`_ where you can easily purchase and manage your Mattermost Enterprise subscriptions.
+
+Licenses purchased via the Customer Portal are stored and listed in the portal, with license details including their start and end dates, providing you with an at-a-glance overview of your account. You also have full access to your billing history  making it easier to manage purchases and renewal dates.
```

Comment:
Consider describing how a license and subscription are related.  "A license is a representation of the subscription you purchased. "